### PR TITLE
Adding fizz_buzz.rb to the repo to be graded.

### PR DESCRIPTION
**Pull request by anthonyfalzetti**
I had to change the puts to print to work correctly. 

[#88505536]

### CODE REVIEW

**Comment by jasonnoble** [comment]
Please add the tracker story # to the commit message.

**Comment by jasonnoble** [comment]
Needs rebasing.

**Comment by anthonyfalzetti** [comment]
rebase complete.

File: methods_and_flow_control/fizz_buzz.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    puts 'Fizz'
+  elsif number % 5 == 0
+    puts 'Buzz'
+  else
+    puts number
+  end
+end
+
+(1..100).each do |number|
+  print fizz_buzz(number)
```
Comment (jasonnoble) [inline methods_and_flow_control/fizz_buzz.rb]: Your method prints things.  Change the method to return something and print it here...

**Pull request merged**